revpi.py

```diff
@@ -126,6 +126,7 @@ def process_one_value(self, name, data):
         Drops faulty temperature measurements, otherwise leaves the conversion from DAC units to something sensible
         to a later function
         """
-        data = int(data)
+        return data # Not sure which faulty readings we are meant to drop, but certainly not all negative ones!
+        data = int(data) # Add a signed=True before wildly uncommenting!
```

Comment:
This part of the code is unreachable. Maybe we delete it? I think the Temperature measurements are sometimes producing spikes where T~6000K that we threw out with line 132 to not mess up our plots. So we could put line 132 before line 129 if we wanted to keep this.

Reply:
Yeah I changed the order to make this part unreachable as a way of commenting it out, should have then followed it up! The problem was we were casting as an unsigned integer and losing negative readings. I suggest we throw it out for now.